### PR TITLE
Sets too long line truncation spec as pending

### PR DESCRIPTION
**Pull request by filipefurtad0**
#### What? Why?

- Relates to #10685

Sets too long line truncation spec as pending.
If approved, can be cherry-picked when fixing #10685 or rebased, after removal of the pending.

#### What should we test?

- Green build.

#### Release notes

Changelog Category: Technical changes

The title of the pull request will be included in the release notes.

### CODE REVIEW

File: spec/system/consumer/shopping/products_spec.rb [hunk 1]
```diff
@@ -47,16 +47,19 @@
...
+                               eget scelerisque nibh auctor non. </p> Link to an '\
                               '<a href="http://google.fr" target="_blank">external site</a>'\
                               '<img src="https://www.openfoodnetwork.org/wp-content/uploads/2019/'\
                               '05/logo-ofn-global-web@2x.png" alt="open food network logo" />'
         product.save!
 
         visit shop_path
         expect(page).to have_content product.name
-
+        expect(page).to have_content "..." # it truncates a long produt description
```
Comment (jibees) [inline spec/system/consumer/shopping/products_spec.rb:62]: I'd add a `withing`, something like (not tested):
```ruby
within "product#product-#{product.id}"
  expect("product-description").to have_content "..."
end
```
to be more specific. 


What do you think?

**Review by jibees — CHANGES_REQUESTED** [review]
I've a request ;) Tell me what do you think ;)

File: spec/system/consumer/shopping/products_spec.rb (hunk 1 above)
Comment (filipefurtad0) [reply spec/system/consumer/shopping/products_spec.rb:62]: Fully agree - looking for three loose dots on the whole page is probably not a very good idea :+1: 

Maybe even adding it within the helper `expect_product_description_html_to_be_displayed` ?
Proposal coming up.

**Comment by jibees** [comment]
@filipefurtad0 

Specs were failing, I've committed to fix that. Tell me what you think ;)

**Comment by filipefurtad0** [comment]
I did not notice that helper was being used in a different test case: setting a `truncate` flag is a great idea. Thanks for that suggestion @jibees , looks great to me.

**Review by jibees — APPROVED** [review]

File: spec/system/consumer/shopping/products_spec.rb [hunk 2]
```diff
@@ -47,16 +47,18 @@
...
-      it "shows HTML product description" do
-        product.description = '<p><b>Formatted</b> product description.</p> Link to an '\
+      it "shows HTML product description and truncates it correctly" do
+        pending "#10685"
+        product.description = '<p><b>Formatted</b> product description: Lorem ipsum dolor sit amet,
+                              consectetur adipiscing elit. Morbi venenatis metus diam,
+                               eget scelerisque nibh auctor non. </p> Link to an '\
                               '<a href="http://google.fr" target="_blank">external site</a>'\
                               '<img src="https://www.openfoodnetwork.org/wp-content/uploads/2019/'\
                               '05/logo-ofn-global-web@2x.png" alt="open food network logo" />'
```
Comment (mkllnk) [inline spec/system/consumer/shopping/products_spec.rb:57]: I would recommend HEREDOC or NOWDOC for bigger blobs of text.

**Review by mkllnk — APPROVED** [review]
Cool.

**Pull request merged**